boxes/application/application.pkr.hcl

```diff
@@ -285,13 +398,42 @@ build {
   }
 
   provisioner "shell" {
-    inline = [
+    execute_command = "sudo bash -c '{{ .Vars }} {{ .Path }}'"
+    inline          = [
       "bash -c '/srv/cray/scripts/common/cleanup-kis-artifacts.sh'"
     ]
     only = ["qemu.application", "virtualbox-ovf.application"]
   }
 
   post-processors {
+    post-processor "shell-local" {
+      # Packer will always make a package.box file from its vagrant builder (https://github.com/hashicorp/packer-plugin-vagrant/issues/16).
+      # This renames package.box to ${source.name}.box, and removes the Vagrantfile since it contains sensitive information (e.g. the root password).
+      inline = [
+        "mv -v ${var.output_directory}-${source.type}/package.box ${var.output_directory}-${source.type}/${var.image_name}.box",
+        "ls -l ${var.output_directory}-${source.type}/"
+      ]
+      only = ["vagrant.application"]
+    }
+
+    # Commented out as no subsequent  layer exists yet 
```

Comment:
@manderson-hpe I commented out this post-processor as we have no further layers (yet). It would have required I change references to `./scripts` to use the vendor/subtree path.